docs/services/navigation/_index.md

```diff
@@ -99,7 +102,7 @@ The following attributes are available for `Navigation` services:
 
 | Name | Type | Inclusion | Description |
 | ---- | ---- | --------- | ----------- |
-| `store` | obj | **Required** | The configuration to use for the Navigation data storage. There are two parameters: `type` (`"memory"` or `"mongodb"`) and `config`. <br> Examples: `"store": [{"type": "memory"}]`, `"store": [{ "type": "mongodb", "config": {"uri": "mongodb://127.0.0.1:27017"}]` |
+| `store` | obj | **Required** | | `store` | obj | **Required** | The type and configuration of data storage to use. Either type `"memory"`, where no additional configuration is needed and the waypoints are stored in local memory while the Navigation process is running, or `"mongodb"`, where data persists at your specified [URI](https://www.mongodb.com/docs/manual/reference/connection-string). |
```

Comment:
I think something broke here - 

```suggestion
| `store` | obj | **Required** | The type and configuration of data storage to use. Either type `"memory"`, where no additional configuration is needed and the waypoints are stored in local memory while the Navigation process is running, or `"mongodb"`, where data persists at the specified [MongoDB URI](https://www.mongodb.com/docs/manual/reference/connection-string) of your MongoDB deployment. |
```

Reply:
@npentrel 🤦‍♀️ Thanks!

Reply:
Also suggested changes to the end of the description :)

Reply:
NVM you got this right